tx-source/site_core.php

```diff
@@ -699,6 +700,7 @@
 $Definition['Group Info'] = 'Group Info';
 $Definition['Group Leaders'] = 'Leaders';
 $Definition['Group Members'] = 'Members';
+$Definition['Groups Menu'] = 'Groups Menu';
```

Comment:
The default value here should be `Groups`. The key is `Groups Menu` so people can change the name if they want.

It even defaults back to `t('Groups')` so we shouldn't don't need to include it again.